galata/README.md

```diff
@@ -12,7 +12,7 @@ Galata is a set of helpers and fixtures for JupyterLab UI Testing using [Playwri
 - **[Rich High Level API](src/jupyterlabpage.ts)** to control and inspect JupyterLab UI programmatically
 - **[Dedicated fixtures](src/fixtures.ts)** to hooks the helpers in the Playwright Page and ensure state isolation between tests.
 
-![screencast](media/screencast.gif)
+![screencast](https://raw.githubusercontent.com/jupyterlab/jupyterlab/master/galata/media/screencast.gif)
```

Comment:
Let's take the opportunity to also remove `screencast.gif` from the repo? It currently weighs ~6 MB and seems to be ending up in the sdist.